crates/astria-sequencer/src/transaction/mod.rs

```diff
@@ -96,6 +98,12 @@ impl ActionHandler for UnsignedTransaction {
                 Action::SudoAddressChange(act) => act
                     .check_stateless()
                     .context("stateless check failed for SudoAddressChangeAction")?,
+                #[cfg(feature = "mint")]
+                Action::Mint(act) => act
+                    .check_stateless()
+                    .context("stateless check failed for MintAction")?,
+                #[cfg(not(feature = "mint"))]
+                _ => bail!("unsupported action type: {:?}", action),
```

Comment:
is this going to crash the sequencer? Sorry, don't remember if action handler errors are printed or bubbled up.

Reply:
no, it won't crash the sequencer - this is used in `deliver_tx` which just warns if a check fails.